src/main/menu/setupBarMenu.js

```diff
@@ -27,7 +27,11 @@ export default () => {
     },
     {
       role: 'window',
-      submenu: [{ role: 'minimize' }, { role: 'close' }],
+      submenu: [
+        { role: 'minimize' },
+        { role: 'close' },
+        { role: 'quit', accelerator: 'CmdOrCtrl+Q' },
```

Comment:
This has to be under a submenu called ”File" and not the window one.

Reply:
@IGassmann like this guys
{
role: 'file',
      submenu: [{ role: 'quit', accelerator: 'CmdOrCtrl+Q' }],
    },

Reply:
or do you mean submenu label : 'file'